tox.ini

```diff
@@ -113,7 +114,10 @@ commands_pre =
   mypyinstalled: pip install file://{toxinidir}/opentelemetry-api/
 
 commands =
-  test: pytest {posargs}
+  test: pytest --benchmark-save=now --benchmark-compare --benchmark-compare-fail=min:75% --benchmark-sort=mean --benchmark-columns=mean,min,max,stddev,median,ops,rounds {posargs}
+
+  core-sdk: {toxinidir}/tests/print_benchmarks.py {toxinidir}/opentelemetry-sdk
```

Comment:
I'm not sure I follow what this is for? Doesn't already print when you run `pytest` in a nice colored format

Reply:
It prints the table which is a nice visual format. However, we want to see the `.json` file produced because `pytest-benchmark` can only run comparisons between `.json` files, not tables.

So the `.json` file is printed to the workflow logs so that the author can see the performance results of their changes according to the normalize standard of "running on GitHub actions".

And if they improved performance they would update the relevant files in `0001_old.json` for each python version 😄  (This is the only manual process of this 😕 )

Reply:
I don't think I understand. They would copy/paste the output JSON from the CI logs into their PR?

Reply:
Close, they would copy/paste the output JSON into the `opentelemetry-sdk/tests/performance/.benchmarks/Linux-CPython-3.5-64bit/0001_old.json` file for example.

This is only if they want to change the expected performance standard, which is something we could automate to happen every "release" for example.